src/components/cylc/table/Table.vue

```diff
@@ -109,6 +109,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             :sort-desc.sync="sortDesc"
             item-key="id"
             show-expand
+            dense
```

Comment:
It might be good to allow the user to choose dense mode or not in Settings or perhaps via a checkbox in the table footer

Reply:
I prefer dense mode, having just tried it.  No harm in having a choice though, I guess.

Reply:
I considered it - I'll see if it's straightforward.

Reply:
Decision: forget about it for now